### PR TITLE
[REF] 8.0 document page procedure

### PR DESCRIPTION
**Pull request by llacroix**
Marked as installable no other change required

Depends on 
- [x] #26
  - [x] OCA/knowledge#30

Edit add dependencies (yvaucher)

### CODE REVIEW

File: document_page_procedure/__openerp__.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "depends": ['document_page_work_instructions'],
     "data": ['document_page_procedure.xml'],
     "demo": [],
-    'installable': False,
+    'installable': True,
     "certificate": ''
 }
 # vim:expandtab:smartindent:tabstop=4:softtabstop=4:shiftwidth=4:
```
Comment (bwrsandman) [inline document_page_procedure/__openerp__.py]: Please remove `#vim` lines

**Comment by bwrsandman** [comment]
Please split `document_page_procedure.xml` into `document_page_data.xml` and `document_page_view.xml`

**Comment by bwrsandman** [comment]
Depends on #26

**Comment by llacroix** [comment]
Splitted and moved icon

File: document_page_procedure/data/document_page.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
```
Comment (dreispt) [inline document_page_procedure/data/document_page.xml]: nopudate="1"?

File: document_page_procedure/views/document_page.xml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+<openerp>
+    <data>
+
+        <record model="ir.actions.act_window" id="action_procedures">
+            <field name="name">Procedures</field>
+            <field name="res_model">document.page</field>
+            <field name="view_type">form</field>
+            <field name="view_mode">tree,form</field>
+            <field name="view_id" ref="document_page.view_wiki_tree"/>
+            <field name="search_view_id" ref="document_page.view_wiki_filter"/>
```
Comment (dreispt) [inline document_page_procedure/views/document_page.xml]: I would remove the two *view_id attributes

**Comment by dreispt** [comment]
Minor notes, but I'm :+1:

File: document_page_procedure/data/document_page.xml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+<h1>Terms and Definitions</h1>
+
+<h1>Responsibilities</h1>
+
+<h1>Process</h1>
+]]>
+        </field>
+      </record>
+
+      <record id="document_page_procedure" model="document.page">
```
Comment (max3903) [inline document_page_procedure/data/document_page.xml]: This record can be removed or moved to a demo data file.

**Comment by dreispt** [comment]
Needs fixing, based on @max3903 comment.

**Comment by yvaucher** [comment]
Dependencies are merged but need fixing

**Comment by llacroix** [comment]
All fixed push

**Comment by max3903** [comment]
Tested on runbot

:+1:

**Pull request merged**